packages/ddp-server/stream_server.js

```diff
@@ -15,8 +15,9 @@ var websocketExtensions = _.once(function () {
   var websocketCompressionConfig = process.env.SERVER_WEBSOCKET_COMPRESSION
         ? JSON.parse(process.env.SERVER_WEBSOCKET_COMPRESSION) : {};
   if (websocketCompressionConfig) {
+    const maxLengthConfig = process.env.WEBSOCKET_FRAME_MESSAGE_MAX_LENGTH;
     extensions.push(Npm.require('permessage-deflate').configure(
-      websocketCompressionConfig
+        { ...websocketCompressionConfig, ...(maxLengthConfig ? { maxLength: parseInt(maxLengthConfig, 10) } : {}) }
```

Comment:
This would look a bit more readable if we conditionally assign `websocketCompressionConfig.maxLength = ...;`, just like it was done for `serverOptions.faye_server_options`.

For consistency also.

Reply:
fixed